### PR TITLE
footer design and logo fixes

### PR DESCRIPTION
**Pull request by tlj4k8**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,10 +10,15 @@
     "heroku-postbuild": "npm run build"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.2",
+    "@fortawesome/free-solid-svg-icons": "^5.2.0",
+    "@fortawesome/vue-fontawesome": "^0.1.1",
     "axios": "^0.18.0",
     "element-ui": "^2.4.1",
     "express": "^4.16.3",
+    "i": "^0.3.6",
```
Comment (dborstelmann) [inline package.json]: ?

File: package.json [hunk 2]
```diff
@@ -10,10 +10,15 @@
...
     "moment": "^2.22.2",
+    "npm": "^6.2.0",
```
Comment (dborstelmann) [inline package.json]: ?

File: package.json [hunk 3]
```diff
@@ -10,10 +10,15 @@
     "heroku-postbuild": "npm run build"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.2",
```
Comment (dborstelmann) [inline package.json]: Do we really need all 3 of these?

File: src/components/Footer.vue [hunk 4]
```diff
@@ -17,73 +15,109 @@
...
-  methods: {
-    about: function() {
-      this.$router.push("/about");
-    }
-  },
   computed: mapState({
-    computedRegister (state){
-      return state.username === '' ? 'registration' : 'profile';
+    computedRegister(state) {
+      return state.username === "" ? "registration" : "profile";
```
Comment (dborstelmann) [inline src/components/Footer.vue]: You really gotta turn your double quote linter off

File: src/components/Footer.vue [hunk 5]
```diff
@@ -17,73 +15,109 @@
...
-}
-.logo {
-  width: 75px;
-}
 .slalomLogo {
   width: 150px;
 }
+.socialLinks {
+  display: block;
+  margin-top: 230px;
```
Comment (dborstelmann) [inline src/components/Footer.vue]: That’s a big margin, what’s the reason for that?

File: src/main.js [hunk 6]
```diff
@@ -8,7 +8,13 @@ import store from "./store";
 import VeeValidate from 'vee-validate';
 import Vuex from 'vuex'
 import axios from 'axios';
+import { library } from '@fortawesome/fontawesome-svg-core'
```
Comment (dborstelmann) [inline src/main.js]: Again, really necessary?  Font awesome base should have all you need

Comment (tlj4k8) [reply src/main.js]: I wasn't sure, when I was look up the steps to get font awesome, it was showing I needed this but I will take it out.

File: src/components/Footer.vue (hunk 5 above)
Comment (tlj4k8) [reply src/components/Footer.vue]: I was using flex and wanted to make sure that the slalom logo was at the bottom of the page. Do you have a better suggestion?

File: package.json (hunk 2 above)
Comment (tlj4k8) [reply package.json]: I'm not sure what this is about.. I only installed the font awesome

File: package.json (hunk 1 above)
Comment (tlj4k8) [reply package.json]: deleted

File: package.json (hunk 2 above)
Comment (tlj4k8) [reply package.json]: deleted

File: package.json (hunk 3 above)
Comment (tlj4k8) [reply package.json]: nope!

File: src/components/Footer.vue (hunk 4 above)
Comment (tlj4k8) [reply src/components/Footer.vue]: Yes I agree

File: src/components/Footer.vue (hunk 5 above)
Comment (dborstelmann) [reply src/components/Footer.vue]: Nah this is fine.  Keep it and we can talk later.

File: src/main.js (hunk 6 above)
Comment (dborstelmann) [reply src/main.js]: This is fort awesome.   Not font awesome.  That’s why I am sketched out.

Comment (dborstelmann) [reply src/main.js]: Installing font awesome should be a tad different

**Pull request merged**